README.md

```diff
@@ -30,42 +30,79 @@ Build modern, graph-based applications on top of your streaming data in minutes.
 </p>
 
 ## :clipboard: Description
-Memgraph is a streaming graph application platform that helps you wrangle your streaming data, build sophisticated models that you can query in real-time, and develop graph applications.
 
-Memgraph directly connects to your streaming infrastructure. You can ingest data from sources like Kafka, SQL, or plain CSV files.
-Memgraph provides a standard interface to query your data with Cypher, a widely-used and declarative query language that is easy to write, understand and optimize for performance.
-This is achieved by using the property graph data model, which stores data in terms of objects, their attributes, and the relationships that connect them.
-This is a natural and effective way to model many real-world problems without relying on complex SQL schemas.
+Memgraph is a streaming graph application platform that helps you wrangle your
+streaming data, build sophisticated models that you can query in real-time, and
+develop graph applications.
 
-Memgraph is implemented in C/C++ and leverages an in-memory first architecture to ensure that you’re getting the best possible performance consistently and without surprises.
-It’s also ACID-compliant and highly available.
+Memgraph directly connects to your streaming infrastructure. You can ingest data
+from sources like Kafka, SQL, or plain CSV files. Memgraph provides a standard
+interface to query your data with Cypher, a widely-used and declarative query
+language that is easy to write, understand and optimize for performance. This is
+achieved by using the property graph data model, which stores data in terms of
+objects, their attributes, and the relationships that connect them. This is a
+natural and effective way to model many real-world problems without relying on
+complex SQL schemas.
+
+Memgraph is implemented in C/C++ and leverages an in-memory first architecture
+to ensure that you’re getting the best possible performance consistently and
+without surprises. It’s also ACID-compliant and highly available.
+
+## :shipit: Installation
```

Comment:
I would replace it with `Download & Install` - explains that you don't have to download code to run it because there is a binary download available (just `Installation` might mean there is a guide on how to run the code)